Description: Drop netbsd and openbsd from cherry pick tests
 Support for netbsd and openbsd is on in upstream until new-upstream-snapshot.
Author: Chad Smith <chad.smith@canonical.com>
Origin: backport
Forwarded: not-needed
Last-Update: 2020-04-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/unittests/test_render_cloudcfg.py
+++ b/tests/unittests/test_render_cloudcfg.py
@@ -9,7 +9,7 @@ from cloudinit import util
 
 # TODO(Look to align with tools.render-cloudcfg or cloudinit.distos.OSFAMILIES)
 DISTRO_VARIANTS = ["amazon", "arch", "centos", "debian", "fedora", "freebsd",
-                   "netbsd", "openbsd", "rhel", "suse", "ubuntu", "unknown"]
+                   "rhel", "suse", "ubuntu", "unknown"]
 
 
 class TestRenderCloudCfg:
@@ -40,18 +40,3 @@ class TestRenderCloudCfg:
             'amazon': 'ec2-user', 'debian': 'ubuntu', 'unknown': 'ubuntu'}
         default_user = system_cfg['system_info']['default_user']['name']
         assert default_user == default_user_exceptions.get(variant, variant)
-
-    @pytest.mark.parametrize('variant,renderers', (
-        ('freebsd', ['freebsd']), ('netbsd', ['netbsd']),
-        ('openbsd', ['openbsd']), ('ubuntu', ['netplan', 'eni', 'sysconfig']))
-    )
-    def test_variant_sets_network_renderer_priority_in_cloud_cfg(
-        self, variant, renderers, tmpdir
-    ):
-        outfile = tmpdir.join('outcfg').strpath
-        util.subp(
-            self.cmd + ['--variant', variant, self.tmpl_path, outfile])
-        with open(outfile) as stream:
-            system_cfg = util.load_yaml(stream.read())
-
-        assert renderers == system_cfg['system_info']['network']['renderers']
